### PR TITLE
make admin page

### PR DESCRIPTION
**Pull request by mariamisa**
-view admin budget
- view artist data
-switch acount artist active

relates #93

### CODE REVIEW

File: client/src/components/Admin/index.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      if (err.response) {
+        setError('حدث خطا في جلب بيانات الادمن');
+      }
+    }
+  };
+  useEffect(() => {
+    getAdminBudget();
+  }, []);
+
+  const getAllArtistData = async () => {
```
Comment (Mu7ammadAbed) [inline client/src/components/Admin/index.js:29]:
```suggestion
  const getAllArtistsData = async () => {
```

File: server/controllers/user/getAdminData.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const { getAdminBudget } = require('../../database/queries/index');
+
+const adminData = async (req, res, next) => {
+  try {
+    const { rows: data } = await getAdminBudget();
+    res.status(200).json({ statusCode: 200, data });
```
Comment (Mu7ammadAbed) [inline server/controllers/user/getAdminData.js:6]:
```suggestion
    res.json({ statusCode: 200, data });
```

File: server/controllers/user/getAdminData.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  } catch (err) {
+    next(err);
+  }
+  res.json({ msg: 'admin data' });
```
Comment (Mu7ammadAbed) [inline server/controllers/user/getAdminData.js:10]: For what this response is sent ?

File: server/database/queries/admin/getAdmin.js [hunk 4]
```diff
@@ -5,5 +5,9 @@ const getAdminEmail = (email) =>
     text: `SELECT * FROM admin WHERE email=$1;`,
     values: [email],
   });
+const getAdminBudget = () =>
```
Comment (Mu7ammadAbed) [inline server/database/queries/admin/getAdmin.js:8]:
```suggestion
const getAdminBudget = () => connection.query('SELECT budget FROM admin WHERE id=1');
```

File: server/controllers/user/getAdminData.js (hunk 3 above)
Comment (mariamisa) [reply server/controllers/user/getAdminData.js:10]: for test controller and forget to delete :)

**Pull request merged**